charts/ceph-csi-rbd/templates/csidriver-crd.yaml

```diff
@@ -1,5 +1,4 @@
----
```

Comment:
this is cephfs file not rbd file.. same for other places also when I deploy rbd from helm chat am seeing cephfs templates inside it please check.

Reply:
This was due the values.yaml containing wrong values, has been fixed in 7761328f5992b213caf73c0c710cb15481767485